collections.md

```diff
@@ -125,6 +125,19 @@ Return codes:
 * 200 OK - if the operation succeed
 * 400 Bad Request - if the uuid parameter is missing or invalid
 
+#### findAdminAuthorized
+**/api/core/collections/search/findAdminAuthorized**
+
```

Comment:
This should have description, something like:

`Returns all Collections in site for which the currently authenticated user has Admin privileges. No Collections are returned if the user does not have Admin privileges.`